bundles/af-core/src/test/java/com/adobe/cq/forms/core/Utils.java

```diff
@@ -125,7 +125,7 @@ public static void testSchemaValidation(@NotNull Object model) {
         // create an instance of the JsonSchemaFactory using version flag
         JsonSchemaFactory schemaFactory = JsonSchemaFactory.getInstance(SpecVersion.VersionFlag.V7);
         try {
-            InputStream schemaStream = Utils.class.getResourceAsStream("/schema/0.12.1/adaptive-form.schema.json");
+            InputStream schemaStream = Utils.class.getResourceAsStream("/schema/0.12.5/adaptive-form.schema.json");
```

Comment:
Can you change the default version of spec in container to 0.12.5 ? https://github.com/adobe/aem-core-forms-components/blob/master/bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java#L67

Reply:
Since this new property is added via expression editor, I believe expression editor would be persisting the `specVersion` property in JCR ?

Reply:
Yes I'm raising the PR for that.